### ISSUE TITLE
Cannot run macro on multiple line visual selection (VsVim 1.4.2)

### ISSUE THREAD

**Issue by constantineiv** [VsVim/VsVim#1210]
A macro has been recorded as:

```
qq<sequence of normal mode commands>q
```

After trying to run macro on multiple selected lines as:

```
:'<,'>norm @q
```

```
":Parse error" is returned.
```

This operation works as expected in gVim 7.4.

**Comment by MaxLevs** [comment]
In 2013, I thought that the inability to run a macro for several lines at once was a big problem. And now I think ... absolutely exactly the same :D

**Comment by ricksladkey** [comment]
This was implemented in PR #1695. If a specific recorded macro does not work correctly when executed as a normal command on a line range, please open a new issue.
